src/gateways/evidence-api.test.ts

```diff
@@ -107,141 +277,146 @@ describe('Evidence api gateway', () => {
   });
 
   describe('POST request to /evidence_requests', () => {
-    const path = ['evidence_requests'];
-    const method = 'POST';
-    const headers = {
-      useremail: Constants.DUMMY_EMAIL,
-    };
-    const body = {
-      data: 'bar',
-    };
-    const expectedData = 'response to POST';
-    const expectedStatus = 200;
-    const expectedResponse = { data: expectedData, status: expectedStatus };
-
-    beforeEach(() => {
-      client.request.mockImplementation(
-        async () =>
-          ({
-            data: expectedData,
-            status: expectedStatus,
-          } as AxiosResponse)
-      );
-    });
+    describe('when the request is successful', () => {
```

Comment:
This bit is just reorganising some of the tests, as their nesting was out of place